prelims/processor/tests/test_recommender.py

```diff
@@ -4,18 +4,15 @@
 from unittest import TestCase
 
 import os
-
+from pathlib import Path
 
 class RecommenderTestCase(TestCase):
 
     def setUp(self):
         # file paths
-        self.path_a = os.path.abspath(
-                os.path.join(os.sep, 'path', 'to', 'articles', 'a.md'))
-        self.path_b = os.path.abspath(
-                os.path.join(os.sep, 'path', 'to', 'articles', 'b.md'))
-        self.path_c = os.path.abspath(
-                os.path.join(os.sep, 'path', 'to', 'articles', 'c', 'index.md'))
+        self.path_a = Path('path').joinpath(f'{os.sep}to', 'articles', 'a.md')
+        self.path_b = Path('path').joinpath(f'{os.sep}to', 'articles', 'b.md')
+        self.path_c = Path('path').joinpath(f'{os.sep}to', 'articles', 'c', 'index.md')
```

Comment:
Do you mean `Path(f'{os.sep}path').joinpath('to', 'articles', 'a.md')`?

```py
>>> from pathlib import Path
>>> import os
>>> Path('path').joinpath(f'{os.sep}to', 'articles', 'a.md')
PosixPath('/to/articles/a.md')
>>> Path(f'{os.sep}path').joinpath('to', 'articles', 'a.md')
PosixPath('/path/to/articles/a.md')
```

Also appreciated if you could update https://github.com/takuti/prelims/blob/master/prelims/processor/tests/test_opengraph.py#L12-L13 accordingly.

Reply:
Fixed 6563918